tests/test_tenant_unauthorized.py

```diff
@@ -0,0 +1,115 @@
+    'Mutation.createQuicksightDataSourceSet',  #  admin action. No need for tenant permission check
+    'Mutation.startMaintenanceWindow',  #  admin action. No need for tenant permission check
+    'Mutation.stopMaintenanceWindow',  #  admin action. No need for tenant permission check
+    'Mutation.startReindexCatalog',  #  admin action. No need for tenant permission check
+    'Mutation.markNotificationAsRead',  #  tenant permissions do not apply to user personal notifications.
+    'Mutation.deleteNotification',  #  tenant permissions do not apply to user personal notifications.
+    'Mutation.postFeedMessage',  #  tenant permissions do not apply to user personal feed comments.
+    'Mutation.upVote',  #  tenant permissions do not apply to user personal up votes.
+    'Mutation.createShareObject',  #  follow-up PR. ADD MANAGE_SHARE permission
+    'Mutation.deleteShareObject',
+    'Mutation.cancelShareExtension',
+    'Mutation.addSharedItem',
+    'Mutation.removeSharedItem',
+    'Mutation.submitShareObject',
+    'Mutation.submitShareExtension',
+    'Mutation.approveShareObject',
+    'Mutation.approveShareExtension',
+    'Mutation.rejectShareObject',
+    'Mutation.revokeItemsShareObject',
+    'Mutation.verifyItemsShareObject',
+    'Mutation.reApplyItemsShareObject',
+    'Mutation.updateShareRejectReason',
+    'Mutation.updateShareExpirationPeriod',
+    'Mutation.updateShareExtensionReason',
+    'Mutation.updateShareRequestReason',
+    'Mutation.updateShareItemFilters',
+    'Mutation.removeShareItemFilter',
+    'Mutation.createAttachedMetadataForm',  #  outside of this PR to be able to backport to v2.6.2
+    'Mutation.deleteAttachedMetadataForm',  #  outside of this PR to be able to backport to v2.6.2
+    'Mutation.createRedshiftConnection',  #  outside of this PR to be able to backport to v2.6.2
+    'Mutation.deleteRedshiftConnection',  #  outside of this PR to be able to backport to v2.6.2
+    'Mutation.addConnectionGroupPermission',  #  outside of this PR to be able to backport to v2.6.2
+    'Mutation.deleteConnectionGroupPermission',  #  outside of this PR to be able to backport to v2.6.2
+]
+
+OPT_IN_QUERIES = [
+    'Query.generateEnvironmentAccessToken',
+    'Query.getEnvironmentAssumeRoleUrl',
+    'Query.getSagemakerStudioUserPresignedUrl',
+    'Query.getSagemakerNotebookPresignedUrl',
+    'Query.getDatasetAssumeRoleUrl',
+    'Query.getDatasetPresignedUrl',
+    'Query.getAuthorSession',
+    'Query.getDatasetSharedAssumeRoleUrl',
+    'Query.runAthenaSqlQuery',
+]
+
+ALL_QUERIES = {
+    f'{_type.name}.{field.name}': field.resolver
+    for _type in bootstrap().types
+    for field in _type.fields
+    if field.resolver and _type.name == 'Query'
+}
+
+ALL_MUTATIONS = {
+    f'{_type.name}.{field.name}': field.resolver
+    for _type in bootstrap().types
+    for field in _type.fields
+    if field.resolver and _type.name == 'Mutation'
+}
+
+
+@pytest.mark.parametrize(
```

Comment:
I think we should have one global variable let's say ALL_RESOLVERS and then filter accordingly on each test and if they don't pass the filter we raise a `pytest.skip(...)`

The reason I suggest that is because
1. We will be able to reuse this variable in other tests that require resolvers
2. It becomes obvious that we run those tests against ALL resolvers but we explicitly skip some

Wdyt?